src/Examples/Flowsheets/ngcc/soec.py

```diff
@@ -1150,34 +994,15 @@ def initialize(
         propagate_state(self.feed01b)
         propagate_state(self.sweep01b)
 
-        self.fuel_inlet_translator.initialize(
-            outlvl=outlvl, solver=solver, optarg=optarg
-        )
-        self.oxygen_inlet_translator.initialize(
-            outlvl=outlvl, solver=solver, optarg=optarg
-        )
-
-        propagate_state(self.fuel_inlet_translator_arc)
-        propagate_state(self.oxygen_inlet_translator_arc)
-
-        self.soec.potential.fix(1.29)
-        self.soec.initialize(
+        self.soec_stack.solid_oxide_cell.potential.fix(1.29)
+        self.soec_stack.initialize(
             outlvl=outlvl,
             solver=solver,
             optarg=optarg,
             current_density_guess=-5000,
             temperature_guess=1023.15,
         )
-        self.soec.potential.unfix()
-        propagate_state(self.fuel_outlet_translator_arc)
-        propagate_state(self.oxygen_outlet_translator_arc)
-
-        self.fuel_outlet_translator.initialize(
-            outlvl=outlvl, solver=solver, optarg=optarg
-        )
-        self.oxygen_outlet_translator.initialize(
-            outlvl=outlvl, solver=solver, optarg=optarg
-        )
+        self.soec_stack.solid_oxide_cell.potential.unfix()
```

Comment:
Should be fixing and unfixing the stack-level reference variable `potential_cell`